DOCS-8558 made edits (#275)
mulesoft/docs-api-manager: modules/ROOT/pages/configure-multiple-credential-providers.adoc

You can use an Anypoint Platform native client provider (default) or configure an external client provider. To assign separate client providers for different organizations and environments, you must first enable client applications to be authorized using OAuth for the client providers that you want to implement in your organization.

> In previous versions of Anypoint Platform, only a single IdP: either an external IdP or the default IdP (Anypoint Platform native client provider) was supported. However, with the multiple client providers feature, Anypoint Platform supports either one default IdP or multiple external IdPs.

Reviewer: > Anypoint Platform supports either one default IdP or multiple external IdPs.

Either the default client provider or one, or more, external IdPs.

Author: Done! Thanks!

> In previous versions of Anypoint Platform, only a single IdP: either an external IdP or the default IdP (Anypoint Platform native client provider) was supported. However, with the multiple client providers feature, Anypoint Platform supports either one default IdP, or one or more external IdPs.

Reviewer 2: To make this text future-proof, consider naming the precise release number:
```suggestion
In Anypoint Platform [x.x] and earlier, only a single IdP was supported: either an external IdP or the default IdP (Anypoint Platform native client provider). With the addition of the multiple client providers feature, Anypoint Platform [x.x] and later supports either one default IdP or one or more external IdPs.
```

Author: Done! Thanks!

* When you change the client provider for an existing API instance, ensure that the API:
** Has no contracts
** Has no client provider-related policies applied, such as PingFederate and OpenID policies

> * If previously did not assign a client provider to an environment when promoting and importing an API, after you assign an external client provider, all existing APIs in that environment still use the native Anypoint Platform client provider. 

Reviewer: I did not understood this phrase, could we rephrase it?

I understand that the goal of the paragraph is to understand that, when you assign a client provider to an environment, all APIs created before such assignment, they will use the default provider. But a new API will get the external IdP assigned.

Author: Done! Thanks!

+
You can then reassign the appropriate external client provider for that API.
* After you assign an external client provider to an API, you cannot return to using the default native Anypoint Platform client provider.
+

> You can user either the default IdP or multiple external IdPs. You cannot combine using an external client provider with the default provider.

Reviewer: or, one, or multiple, IdPs.

Author: Done! Thanks!

* When you change the client provider for an existing API instance, ensure that the API:
** Has no contracts
** Has no client provider-related policies applied, such as PingFederate and OpenID policies

> * If you previously did not assign an external client provider to an environment when promoting and importing an API, the native provider is used by default. 

Reviewer 2: active voice
```suggestion
* If you previously did not assign an external client provider to an environment when promoting and importing an API, Anypoint Platform [x.x] and later uses the native provider by default. 
```

Author: Done! Thanks!

+
You can then reassign the appropriate external client provider for that API.

> * If you assign an external client provider to an environment that was previously using a native provider, all existing APIs in that environment still use the native Anypoint Platform client provider. The new external IdP is used only by new APIs.

Reviewer 2: This might break up a complicated sentence:
```suggestion
* If you assign an external client provider to an environment that was previously using a native provider: +
* Existing APIs in that environment continue to use the native Anypoint Platform client provider. 
* New API use the new external IdP.
```
OR
```suggestion
* If you assign an external client provider to an environment that was previously using a native provider, only new external APIs use that external IdP.
```

Author: Done! Thanks!

* After you assign an external client provider to an API, you cannot return to using the default native Anypoint Platform client provider.
+

> You can user either the default IdP, or one or more external IdPs. You cannot combine using an external client provider with the default provider.

Reviewer 2: ```suggestion
You can user either the default IdP or one or more external IdPs. You cannot use the default provider and an external client provider together.
```

Author: Done! Thanks!
